tests/Doctrine/Tests/DBAL/Query/Expression/ExpressionBuilderTest.php

```diff
@@ -45,11 +45,7 @@ public function testAnd(array $parts, string $expected) : void
      */
     public function testAndX(array $parts, string $expected) : void
     {
-        $composite = $this->expr->andX();
-
-        foreach ($parts as $part) {
-            $composite->add($part);
-        }
+        $composite = $this->expr->andX()->with(...$parts);
```

Comment:
Since this test is about the legacy API, should we leave it untouched to guarantee the old API still works?

Reply:
Makes sense, I guess. Fixed in 19a850de13858ba4dbc1d7d9bb3386883a4205e5.